src/storage-client/src/controller.rs

```diff
@@ -2913,6 +2873,56 @@ where
                 .await;
         }
     }
+
+    /// Converts an `IngestionDescription<()>` into `IngestionDescription<CollectionMetadata>`.
+    fn describe_ingestion(
```

Comment:
just musing: maybe `populate_ingestion`, or `enrich_ingestion`? But you're better with words than I am... 😅

Reply:
enrich it is